From 3f4eff598c86f7ded9f09fe00a32d79cae1cb6b9 Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Fri, 10 Feb 2017 16:49:43 -0800
Subject: [PATCH] joyent/node-triton#170 Integration tests fail to change
 company name back to original value when it contains spaces Reviewed by: Todd
 Whiteman <todd.whiteman@joyent.com> Approved by: Todd Whiteman
 <todd.whiteman@joyent.com>

---
 test/integration/cli-account.test.js | 15 +++++++++------
 1 file changed, 9 insertions(+), 6 deletions(-)

diff --git a/test/integration/cli-account.test.js b/test/integration/cli-account.test.js
index 04b42b1..ff0cf33 100644
--- a/test/integration/cli-account.test.js
+++ b/test/integration/cli-account.test.js
@@ -13,6 +13,7 @@
  */
 
 var h = require('./helpers');
+var format = require('util').format;
 var test = require('tape');
 
 
@@ -100,11 +101,13 @@ test('triton account', function (tt) {
 
     tt.test(' triton account update companyName=<oldvalue>', writeTestOpts,
             function (t) {
-        h.triton('account update companyName=' + account.companyName || '',
-                function (err, _o, _e) {
-            if (h.ifErr(t, err))
-                return t.end();
-            t.end();
-        });
+        h.triton(
+            format('account update companyName=\'%s\'',
+                account.companyName || ''),
+            function (err, _o, _e) {
+                if (h.ifErr(t, err))
+                    return t.end();
+                t.end();
+            });
     });
 });
-- 
2.21.0

